assets/javascript/app.js

```diff
@@ -0,0 +1,93 @@
+// namespace object for food-finding functions
+var rest_test = {
+  
+  restaurants: [{
+    name: "McDonald's",
+    image:"url",
+    foodtype:"American",
+    description:"fast food",
+    link:"url"
+  }],
+
+  getNearbyRestaurants: function() {
+    return restaurants[Math.floor(Math.random()*restaurants.length)];
+  },
+
+  setFoodClick: function() {
+    $("#findfood").click(function() {
+      var resultsBody = $("<div>");
+  
+      var resultsName = $("<h2>").text(getNearbyRestaurants().name);
+      resultsName.addClass("");
+  
+      var resultsImage = $("<p>").text(getNearbyRestaurants().image);
+      resultsImage.addClass("");
+  
+      var resultsFood = $("<p>").text(getNearbyRestaurants().foodtype);
+      resultsFood.addClass("");
+  
+      var resultsDescription = $("<p>").text(getNearbyRestaurants().description);
+      resultsDescription.addClass("");
+  
+      var resultsLink = $("<p>").text(getNearbyRestaurants().link);
+      resultsLink.addClass("");
+  
+      resultsBody.append(resultsName, resultsImage, resultsFood, resultsDescription, resultsLink);
+      $('#results').append(resultsBody);
+    });
+  },
+
+  getNearbyLoc: function() {
```

Comment:
I mentioned this on @nguyenwinle's review, but now that we have a functional getNearbyRestaurants function, I think we can get rid of this.

Reply:
Agreed.